README.md

```diff
@@ -1,2 +1,4 @@
 # testyMcTestFace
-dsgdsgfgs
+
+
+A proper desc
```

Comment:
/ticket Do that other thing -d Because of reasons -t bug

Reply:
Here's your ticket: [ Do that other thing](https://ticketgen19.atlassian.net/browse/IKKI-50)

Reply:
/ticket Take 2

Reply:
Here's your ticket: [ Take 2](https://ticketgen19.atlassian.net/browse/IKKI-51)

Reply:
Here's your ticket: [ Take 2](https://ticketgen19.atlassian.net/browse/IKKI-52)

Reply:
/ticket Hey -d uhy -t bug

Reply:
Here's your ticket: [ Hey](https://ticketgen19.atlassian.net/browse/IKKI-53)